### PR TITLE
[FORMS-9411] implement plain text/ title/ image component

### PR DESCRIPTION
**Pull request by Aabha1**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: packages/material-ui-based-components/src/components/PlainText.tsx [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+import { withRuleEngine, richTextString } from '../shared/withRuleEngine';
+import { PROPS } from '../utils/types';
+import { Typography } from '@mui/material';
+
+
+const PlainTextComponent = (props: PROPS) => {
+    const { value } = props;
+
+    return (
+        <Typography gutterBottom>{richTextString(value)}</Typography>
```
Comment (amit-mnnit) [inline packages/material-ui-based-components/src/components/PlainText.tsx:11]: please use richText property for rendering rich text or not

File: packages/material-ui-based-components/src/components/Title.tsx [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+import { withRuleEngine, richTextString } from '../shared/withRuleEngine';
+import { PROPS } from '../utils/types';
+import { Typography } from '@mui/material';
+
+
+const TitleComponent = (props: PROPS) => {
+    const { text } = props;
+
+    return (
+        <Typography gutterBottom>{richTextString(text)}</Typography>
```
Comment (amit-mnnit) [inline packages/material-ui-based-components/src/components/Title.tsx:11]: please use richText property for rendering rich text or not

File: packages/material-ui-based-components/src/utils/mappings.ts [hunk 3]
```diff
@@ -1,9 +1,13 @@
...
+import PlainText from '../components/PlainText';
+import Title from '../components/Title';
 
 export default {
   form: Form,
   panel: Panel,
-  'text-input': TextField
+  'text-input': TextField,
+  'plain-text': PlainText,
+  title: Title
```
Comment (amit-mnnit) [inline packages/material-ui-based-components/src/utils/mappings.ts:12]: custom:title

**Review by amit-mnnit — CHANGES_REQUESTED** [review]
Please incorporate all the comments and also test localization

File: packages/material-ui-based-components/src/components/Title.tsx (hunk 2 above)
Comment (Aabha1) [reply packages/material-ui-based-components/src/components/Title.tsx]: added richText property

File: packages/material-ui-based-components/src/utils/mappings.ts (hunk 3 above)
Comment (Aabha1) [reply packages/material-ui-based-components/src/utils/mappings.ts]: done

**Pull request merged**